### PR TITLE
Enabling skins to define a parent skin

### PR DESCRIPTION
**Pull request by hellopablo**
Child skins can selectively override aspects of the parent skin, e.g styles and or views.

Also:
- Updated variable naming convention
- A few files were still using tabs
- A bug fix or two

### CODE REVIEW

File: blog/controllers/blog.php [hunk 1]
```diff
@@ -307,7 +317,7 @@ public function category()
         // --------------------------------------------------------------------------
 
         $this->load->view('structure/header', $this->data);
-        $this->load->view($this->_skin->path . 'views/browse', $this->data);
+        $this->loadView('browse', $this->data);
```
Comment (gsdd) [inline blog/controllers/blog.php:320]: Typo

File: blog/controllers/blog.php [hunk 2]
```diff
@@ -405,7 +415,7 @@ public function tag()
...
```
Comment (gsdd) [inline blog/controllers/blog.php]: Typo

File: blog/controllers/blog.php [hunk 3]
```diff
@@ -441,7 +451,7 @@ public function rss()
 
         //  Set Output
         $this->output->set_content_type('text/xml; charset=UTF-8');
-        $this->load->view($this->_skin->path . 'views/rss', $this->data);
+        $this->loadView('rss', $this->data);
```
Comment (gsdd) [inline blog/controllers/blog.php]: Another one

File: blog/controllers/blog.php (hunk 1 above)
Comment (hellopablo) [inline blog/controllers/blog.php:320]: Nope. New method called loadView().

**Pull request merged**